src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
```

Comment:
You are using `Movie` table here.
You can add @Table annotation so table naming is consistent with the guidelines